### PR TITLE
Log Resource Access in Debug Mode 

### PR DESCRIPTION
**Pull request by fheuschkel**
#127 Log output implemented in the style of HTTP request commands.

### CODE REVIEW

File: modules/interaction/src/blaze/interaction/history/instance.clj [hunk 1]
```diff
@@ -72,6 +72,11 @@
   (fn [{::reitit/keys [router match] :keys [query-params]
         {{:fhir.resource/keys [type]} :data} ::reitit/match
         {:keys [id]} :path-params}]
+    (log/debug
+      (if query-params
+        (format "Get [base]/%s/%s/_history?%s" type id query-params)
```
Comment (alexanderkiel) [inline modules/interaction/src/blaze/interaction/history/instance.clj:77]: Get needs to be uppercase.  query-params need to be formatted in the usual URL style: 
```
k1=v1&k2=v2
```
You can use `clojure.string/join` here.

File: modules/interaction/src/blaze/interaction/history/system.clj [hunk 2]
```diff
@@ -69,6 +69,9 @@
 
 (defn- handler-intern [node]
   (fn [{::reitit/keys [router match] :keys [query-params]}]
+    (if query-params
```
Comment (alexanderkiel) [inline modules/interaction/src/blaze/interaction/history/system.clj:72]: `log/debug` is missing here.

File: modules/interaction/src/blaze/interaction/history/type.clj [hunk 3]
```diff
@@ -69,6 +69,9 @@
 (defn- handler-intern [node]
   (fn [{::reitit/keys [router match] :keys [query-params]
         {{:fhir.resource/keys [type]} :data} ::reitit/match}]
+    (if query-params
```
Comment (alexanderkiel) [inline modules/interaction/src/blaze/interaction/history/type.clj:72]: `log/debug` missing

File: modules/interaction/src/blaze/interaction/read.clj [hunk 4]
```diff
@@ -50,6 +50,11 @@
 (defn- handler-intern [node]
   (fn [{{{:fhir.resource/keys [type]} :data} ::reitit/match
         {:keys [id vid]} :path-params}]
+    (log/debug
+      (if vid
+        (format "Get [base]/%s/%s/_history/%s" type id vid)
```
Comment (alexanderkiel) [inline modules/interaction/src/blaze/interaction/read.clj:55]: Get

File: modules/interaction/src/blaze/interaction/read.clj [hunk 5]
```diff
@@ -50,6 +50,11 @@
...
+        (format "GET [base]/%s/%s" type id))
+      )
```
Comment (alexanderkiel) [inline modules/interaction/src/blaze/interaction/read.clj:57]: the parenthesis should not be not on a new line

File: modules/interaction/src/blaze/interaction/transaction.clj [hunk 6]
```diff
@@ -334,6 +334,7 @@
 (defn- handler-intern [node executor]
   (fn [{{:keys [type] :as bundle} :body :keys [headers]
         ::reitit/keys [router match]}]
+    (log/debug (format "POST [base]"))
```
Comment (alexanderkiel) [inline modules/interaction/src/blaze/interaction/transaction.clj:337]: `format` is not needed here

File: modules/interaction/src/blaze/interaction/history/instance.clj (hunk 1 above)
Comment (fheuschkel) [reply modules/interaction/src/blaze/interaction/history/instance.clj]: Done!

File: modules/interaction/src/blaze/interaction/history/system.clj (hunk 2 above)
Comment (fheuschkel) [reply modules/interaction/src/blaze/interaction/history/system.clj]: Corrected

File: modules/interaction/src/blaze/interaction/history/type.clj (hunk 3 above)
Comment (fheuschkel) [reply modules/interaction/src/blaze/interaction/history/type.clj]: Corrected

File: modules/interaction/src/blaze/interaction/read.clj (hunk 4 above)
Comment (fheuschkel) [reply modules/interaction/src/blaze/interaction/read.clj]: Corrected

File: modules/interaction/src/blaze/interaction/read.clj (hunk 5 above)
Comment (fheuschkel) [reply modules/interaction/src/blaze/interaction/read.clj]: Corrected

File: modules/interaction/src/blaze/interaction/transaction.clj (hunk 6 above)
Comment (fheuschkel) [reply modules/interaction/src/blaze/interaction/transaction.clj]: Corrected

File: modules/interaction/src/blaze/interaction/history/instance.clj [hunk 7]
```diff
@@ -14,7 +14,8 @@
     [reitit.core :as reitit]
     [ring.middleware.params :refer [wrap-params]]
     [ring.util.response :as ring]
-    [taoensso.timbre :as log]))
+    [taoensso.timbre :as log]
+    [clojure.string :as str]))
```
Comment (alexanderkiel) [inline modules/interaction/src/blaze/interaction/history/instance.clj:18]: Please order the requires lexicographically. Unfortunately Cursive is not doing this automatically in the moment.

File: modules/interaction/src/blaze/interaction/history/system.clj [hunk 8]
```diff
@@ -13,7 +13,8 @@
     [reitit.core :as reitit]
     [ring.middleware.params :refer [wrap-params]]
     [ring.util.response :as ring]
-    [taoensso.timbre :as log]))
+    [taoensso.timbre :as log]
+    [clojure.string :as str]))
```
Comment (alexanderkiel) [inline modules/interaction/src/blaze/interaction/history/system.clj:17]: Same here

File: modules/interaction/src/blaze/interaction/history/type.clj [hunk 9]
```diff
@@ -13,7 +13,8 @@
     [reitit.core :as reitit]
     [ring.middleware.params :refer [wrap-params]]
     [ring.util.response :as ring]
-    [taoensso.timbre :as log]))
+    [taoensso.timbre :as log]
+    [clojure.string :as str]))
```
Comment (alexanderkiel) [inline modules/interaction/src/blaze/interaction/history/type.clj:17]: Same here

**Pull request merged**